app/domain/authentication/authn_jwt/validate_uri_based_parameters.rb

```diff
@@ -0,0 +1,55 @@
+module Authentication
+  module AuthnJwt
+
+    ValidateUriBasedParameters ||= CommandClass.new(
+      dependencies: {
+        # ValidateWebserviceIsWhitelisted calls ValidateAccountExists
+        # we call there ValidateAccountExists for better readability an safety
+        validate_account_exists: ::Authentication::Security::ValidateAccountExists.new,
+        validate_webservice_is_whitelisted: Security::ValidateWebserviceIsWhitelisted.new
+      },
+      inputs: %i[authenticator_input enabled_authenticators]
+    ) do
+
+      # authenticator_input attributes
+      #  :account - from uri - has explicit validation
+      #  :authenticator_name - from uri - has implicit validation with service_id
```

Comment:
what do you mean here? where is this validated?

Reply:
According my understanding when `webservice` resource is created it automatically "bounded" to DAL means if `authenticator_name` and `service_id` does not exist in DB resource creation will failed.
@orenbm Please fix me if I'm wrong.

Reply:
i don't know off by heart, please verify. also - in such a case, what error is written to the log? is it clear?